src/app.rs

```diff
@@ -222,8 +221,14 @@ pub fn App(cx: Scope) -> impl IntoView {
             <main class="flex-1 overflow-y-auto">
               <div class="mx-auto max-w-7xl px-4 pt-8 sm:px-6 lg:px-8">
                 <div class="flex">
-                  <h1 class="flex-1 text-2xl font-bold text-gray-900 dark:text-gray-100">
-                    "Live unconfirmed inscriptions"
+                  <h1 class="flex-2 text-2xl font-bold text-gray-900 dark:text-gray-100">
```

Comment:
nit: Move `text-2xl font-bold text-gray-900 dark:text-gray-100` to `class` of parent `div` to avoid duplication.